Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,262 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defineProperties'
+    ], function(
+        Color,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * Provides access to a feature's properties stored in the 3D tile's batch table, as well
+     * as the ability to show/hide a feature and change its highlight color via
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color}, respectively.
+     * <p>
+     * Modifications to a <code>Cesium3DTileFeature</code> object have the lifetime of the tile's
+     * content.  If the tile's content is unloaded, e.g., due to it going out of view and needing
+     * to free space in the cache for visible tiles, listen to the DOC_TBA event to save any
+     * modifications.
+     * </p>
+     * <p>
+     * Do not construct this directly.  Access it through {@link Cesium3DTileContent#getFeature}
+     * or picking using {@link Scene#pick} and {@link Scene#pickPosition}.
+     * </p>
+     *
+     * @alias Cesium3DTileFeature
+     * @constructor
+     *
+     * @example
+     * // On mouse over, display all the properties for a feature in the console log.
+     * handler.setInputAction(function(movement) {
+     *     var feature = scene.pick(movement.endPosition);
+     *     if (Cesium.defined(feature) && (feature.primitive === tileset)) {
+     *         var properties = tileset.properties;
+     *         if (Cesium.defined(properties)) {
+     *             for (var name in properties) {
+     *                 if (properties.hasOwnProperty(name)) {
+     *                     console.log(name + ': ' + feature.getProperty(name));
+     *                 }
+     *             }
+     *         }
+     *     }
+     * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+     */
+    function Cesium3DTileFeature(tileset, content, batchId) {
+        this._content = content;
+        this._batchId = batchId;
+        this._color = undefined;  // for calling getColor
```

Comment:
The only way I can think of is having a `getColor` function that takes a result, which isn't great from an API perspective.